jscomp/core/js_pass_get_used.ml

```diff
@@ -48,30 +48,31 @@ let post_process_stats my_export_set (defined_idents : J.variable_declaration Ha
     since in this case it can not be global?  
 
 *)
+let super = Js_record_iter.iter  
 let count_collects 
   (* collect used status*)
   (stats : int Hash_ident.t)
   (* collect all def sites *)
```

Comment:
check if this visitor is used a lot, some micro-optimizations could be applied here